apps/explorer/src/modules/addresses/components/TableAddressTokensRow.vue

```diff
@@ -52,31 +64,58 @@ export default class TableAddressTokensRow extends Mixins(StringConcatMixin) {
   ===================================================================================
   */
 
-  @Prop(Object) token!: any
+  @Prop(Object) token!: TokenPageExt_items
   @Prop(String) holder!: string
 
   /*
   ===================================================================================
-    Methods
+    Computed Values
   ===================================================================================
   */
 
-  balance(value, decimals) {
-    const n = new BN(value)
-    return n
-      .div(new BN(10).pow(decimals))
-      .toFixed()
-      .toString()
+  get balance(): string {
+    return this.token.formattedBalance
   }
 
-  /*
-  ===================================================================================
-    Computed Values
-  ===================================================================================
-  */
-
   get tokenLink(): string {
-    return `/token/${this.token.addr}?holder=${this.holder}`
+    return `/token/${this.token.address}?holder=${this.holder}`
+  }
+
+  get currPrice(): string {
+    return this.token.currentPriceBN ? this.getRoundNumber(this.token.currentPriceBN) : '0.00'
+  }
+
+  get  usdValue(): string {
+    return this.getRoundNumber(this.token.usdValueBN)
+  }
+
+  get changeInPrice(): string {
```

Comment:
This seemed specific to the component so I didn't move it to the type class. But if in future we need to use it in multiple components we could move it to TokenPageExt_items class instead.

Reply:
This actually can be good to add it tot he ext_item. This is used on tokens' page, and in token detail page.